backend/src/main/java/com/benine/backend/http/PresetsHandler.java

```diff
@@ -33,6 +36,7 @@ public PresetsHandler() {
     addHandler("recallpreset", new RecallPresetHandler());
     addHandler("addtag", new AddTagHandler());
     addHandler("removetag", new RemoveTagHandler());
+    addHandler("editpreset", new EditPresetHandler());
```

Comment:
If these things are behind the /preset/... endpoint I think edit would suffice, you would get /preset/edit or something similar. Even better would be /preset/[id]/edit to keep it consistent with the camera endpoints.